script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -120,142 +121,144 @@ No, that's not right.
 [sha:fd7d1170d7f2c482b7b9c4ca7c77c0cbac77c080]{
 -- Page 5, Offset 28994.
 -- この両手は、倒れている子供の血で赤いだけ。
--- TRANSLATION HERE
+My hands are red from the collapsed child's blood.
```

Comment:
(to match suggestion)
My hands are just stained red from the blood of the collapsed child.